contributing/code/patches.rst

```diff
@@ -167,8 +181,8 @@ uses, and replaces them by symbolic links to the ones in the Git repository.
 Before running the ``link`` command, be sure that the dependencies of the project you
 want to debug are installed by running ``composer install`` inside it.
 
-Work on your Patch
-~~~~~~~~~~~~~~~~~~
+Work on your PR
```

Comment:
In the text it makes sense to refer to PR, in the title I think it looks better to refer to a "Pull Request" though. It doesn't actually bloat anything and works better when creating a list based on the page titles, as PR is not explained in that list. I think the same goes for all other titles as well